packages/playground/src/data/counterfactual.ts

```diff
@@ -30,12 +30,21 @@ export interface NodeConfig {
 }
 
 export default class CounterfactualNode {
+  private static port: MessagePort;
   private static node: Node;
+  // @ts-ignore
+  private static nodeProvider: cfWallet.NodeProvider;
```

Comment:
❓ Why are we using `@ts-ignore`?

Reply:
Since we're using the bundled version of cf-wallet, ts doesn't have access to the relevant types (NodeProvider and Provider). It errors here because it doesn't recognize `cfWallet`, which might also be expressed as `window.cfWallet` (which also would error). Are you familiar w/ a way around this?

Reply:
Just wanted to confirm that was the case!